extension/src/popup/views/ManageAssets/index.tsx

```diff
@@ -15,34 +16,28 @@ import { AddToken } from "popup/components/manageAssets/AddToken";
 import { PrivateKeyRoute } from "popup/Router";
 import { ROUTES } from "popup/constants/routes";
 
-import { getNetworkDetails } from "background/helpers/account";
-
 export const ManageAssets = () => {
   const {
     accountBalances,
     destinationBalances,
     assetSelect,
     error,
   } = useSelector(transactionSubmissionSelector);
+  const { networkPassphrase } = useSelector(settingsNetworkDetailsSelector);
   const [errorAsset, setErrorAsset] = useState("");
 
-  React.useEffect(() => {
-    const parseXdr = async () => {
-      const { networkPassphrase } = await getNetworkDetails();
```

Comment:
This was throwing an error when running locally in the browser on localhost. `getNetworkDetails` relied on accessing chrome.storage to get the network info. This API, however, doesn't exist in the browser. So, in the `popup`, we should rely on `background` to grab stuff from `chrome.storage` and transmit it to `popup`.

This works fine in production because `popup` gets packaged into the extension and thus has access to chrome.storage. But we should structure our code in a way that works in both the extension and in the browser.

Reply:
Not sure if this worked before and maybe we didn't notice? 🤷

Reply:
ah interesting, but feels weird that we wouldn't have seen this in dev right? Either way this lgtm.